src/applications/coronavirus-vaccination/tests/contract/apply.pact.spec.js

```diff
@@ -0,0 +1,109 @@
+import contractTest from 'platform/testing/contract';
+// import sinon from 'sinon';
+// import { expect } from 'chai';
+import {
+  // decimal,
+  // eachLike,
+  // integer,
+  like,
+  // string,
+  term,
+} from '@pact-foundation/pact/dsl/matchers';
+import { apiPostRequest } from '../../apiCalls/';
+import environment from 'platform/utilities/environment';
+import authenticatedApplicationData from '../cypress/fixtures/data/authenticated-coronavirus-vaccination-application.json';
+import unauthenticatedApplicationData from '../cypress/fixtures/data/unauthenticated-coronavirus-vaccination-application.json';
+
+contractTest('coronavirus-vaccination', 'VA.gov API', mockApi => {
```

Comment:
This needs to be named "Coronavirus Vaccination" to match the app name. It's also the [name used to set up the provider states](https://github.com/department-of-veterans-affairs/vets-api/pull/5466/files#diff-620a6a4731d2a26acd9a98334f53d537e9261ba0c3e7681c3fcfad1766dab2f4R3).

Reply:
Oh, yes! Thanks for that catch. I updated the `appName` but forgot to update it here.